mne/dipole.py

```diff
@@ -445,11 +470,22 @@ def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
     """
     # This could eventually be adapted to work with other inputs, these
     # are what is needed:
+
+    evoked.info['projs'] = []  # get rid of SSP projections
```

Comment:
indeed don't get rid of all projections. Just add the EEG avg ref if not present.

@Eric89GXL to me we should add it automatically. Warning is not enough to make sure people are doing it right.